app/views/declaration/goods_location.scala.html

```diff
@@ -51,14 +51,13 @@
         @errorSummary(form.errors)
 
         @sectionHeader(messages("declaration.summary.locations.header"))
-        @pageTitle(messages("declaration.goodsLocation.title"))
 
         @exportsInputText(
         field = form("code"),
-        labelKey = "",
+        labelKey = "declaration.goodsLocation.title",
         hintKey = Some("declaration.goodsLocation.hint"),
         isPageHeading = true,
-        headingClasses = "govuk-label--m"
+        headingClasses = "govuk-label--xl"
```

Comment:
Can you use the constant `gdsPageLabel` here instead of "govuk-label--xl". (You'll need an extra import - 

@import views.components.gds.Styles._

That "xl" style you have used is the right one according to the prototype but it's larger than the other non-GDS pages.   By using the constant value consistently everywhere, we can hopefully change to "xl" once all the pages have been converted.

Reply:
noted